janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/rest/RestElasticSearchClient.java

```diff
@@ -164,7 +170,7 @@ public void createMapping(String indexName, String typeName, Map<String,Object>
 
     @Override
     public Map getMapping(String indexName, String typeName) throws IOException{
-        Response response = performRequest("GET", "/" + indexName.toLowerCase() + "/_mapping/"+typeName, null);
+        Response response = performRequest("GET", "/" + indexName + "/_mapping/" + typeName, null);
```

Comment:
I see GET appear a few times. Please create string constants for all the HTTP verbs and reuse them.